guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc

```diff
@@ -22,7 +22,6 @@ You can also register Atomic Hosts to {ProjectServer} or {SmartProxyServer}.
 
 Use one of the following procedures to register a host:
 
-* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]
```

Comment:
However this is not valid, your PR is for the `master` branch and in there, this is a valid link.

If you need this to be fixed for Satellite 6.10, then file a PR against 2.5 branch, not master. We cannot merge this just like that.

Reply:
Thanks, understood.